code_challenges.rb

```diff
@@ -9,28 +9,68 @@
 letter_t = 't'
 # Expected output: ['tea', 'water', 'soda water']
 
+#creating a method block
+#trying to use the select method to filter through the array 
+#i kept getting a syntax error im pretty sure i have the logic right but im just not writing it correctly if i could get some input.
+def get_letter 
```

Comment:
yeah, you're getting close but you haven't passed in a parameter (array) so when you call on array.select the method has no idea what you're talking about.  Check your syntax when you're reviewing your code.